src/store/migrations.js

```diff
@@ -41,10 +45,38 @@ export function migrateToReduxStorage(state: any) {
   return parsedState;
 }
 
+export function migrateToSupportProjectHomePath(state: any) {
```

Comment:
thanks for adding this!! I didn't expect you to :) but it's super appreciated.

I think it's also super important to add some tests for migrations, to make sure that when clients download the new app, things work accordingly. I'm happy to tackle this myself, but if you wanted to take a shot at it as well, you can use the existing migration tests as an example